modules/exploits/linux/local/pkexec.rb

```diff
@@ -66,6 +66,20 @@ def executable_path
     @executable_path
   end
 
+	def check
+		version = cmd_exec('pkexec --version')
+		if version.nil?
+			print_error "Policykit is not installed"
+			return CheckCode::Safe
+		end
+
+		if version.split('.')[1].to_f <= 0.101
+			return CheckCode::Vulnerable
```

Comment:
`CheckCode::Appears` per https://github.com/rapid7/metasploit-framework/wiki/How-to-write-a-check()-method